css/style/toast.css

```diff
@@ -0,0 +1,41 @@
+#toast {
+    visibility: hidden;
+    min-width: 250px;
+    margin-left: -125px;
```

Comment:
Avoid using px

Reply:
Thank you for pointing that out That was by mistake. Corrected it. :)